jaeger-core/src/main/java/io/jaegertracing/JaegerTracer.java

```diff
@@ -536,6 +540,11 @@ public Builder withExpandExceptionLogs() {
       return this;
     }
 
+    public Builder withLoggingStackTraces() {
```

Comment:
LoggingStackTraces name might be confused with some more serious feature. I would rename to `withLogStackTraceOnWarnings`.

Also add javadoc explaining the option.